### ISSUE TITLE
Mega sync not working great

### ISSUE THREAD

**Issue by OmaigatTER** [meganz/MEGAsync#1021]
I have mega sync on windows and lately i am having problems with files that arent begin sincronized correctly without even giving me any error or notification I noticed that on every update the mega sync experience begins to degrade over and over this only frustates people to stop buying mega expensive plan .

**Comment by vtmateos** [comment]
Hi,

Hi @OmaigatTER ,

Thank you for your valuable feedback.

To help us resolve your issue, we recommend reaching out to our support team via the “Report Issue” feature. Please explain your case in detail and attach the relevant log files so we can thoroughly analyze the problem. Sharing private information and logs on a public GitHub thread is challenging, so reaching out directly to support will allow us to address your issue more effectively. Once you have a ticket reference number, feel free to share it here, and we’ll help track the progress of your case.

Best regards.

**Comment by Lucretia** [comment]
Yup, I had this issue and reported directly to them. But because I'm on Gentoo it feels like they don't really care.

**Comment by Mrgaton** [comment]
I'm tired already I'm going to switch to icloud wich is cheaper for me sorry but I can't handle it more.

**Comment by Lucretia** [comment]
I just wrote an ebuild to install the gentoo build and it seems to be working.

**Comment by Lennart1978** [comment]
On Archlinux megasync app under Gnome / Wayland is not useable. It always crashes, has problems with file permissions and wants a desktop tray !! On Gnome v.47.  by the way. I deinstalled it and will never come back. A year ago I had the same problems and even after a year of updates nothing works.
